pyvista/utilities/fileio.py

```diff
@@ -233,3 +251,57 @@ def read_exodus(filename,
 
     reader.Update()
     return pyvista.wrap(reader.GetOutput())
+
+
+def read_meshio(filename, file_format = None):
+    """Read any mesh file using meshio."""
+    # Import meshio
+    import meshio
+    from meshio._vtk import (
+        meshio_to_vtk_type,
+        vtk_type_to_numnodes,
+    )
+    
+    # Read mesh file
+    mesh = meshio.read(filename, file_format)
```

Comment:
Wait, if file_format is `None` from the default argument here, will `meshio` automatically chose a reader based on the `_extension_to_filetype` dict? If so, we don't need to use that dict anywhere on PyVista's side

Reply:
Yes, that's right. ``meshio`` also has its own format guessing feature if not provided. Probably better to not use it in ``pyvista.read``.

Reply:
Okay, let's completely rely on `meshio`'s format guessing technique and remove PyVista's use of the `_extension_to_filetype`